__tests__/app.test.js

```diff
@@ -41,4 +41,40 @@ describe('GET /api', () => {
     })
     })
    
-})
\ No newline at end of file
+})
+
+describe('GET /api/reviews/:review_id', () => {
+    test('/api/reviews/2 - status 200 - responds with an object containing 9 properties', ()=> {
+        return request(app).get('/api/reviews/2').expect(200).then(result => {
+            expect(Object.keys(result.body.review[0]).length).toBe(9)
+        })
+    })
+    test('/api/reviews/2 - status 200 - responds with an object containing 9 properties (review_id, title, review_body, designer, review_img_url, votes, category, owner and created_at)', ()=> {
+        return request(app).get('/api/reviews/2').expect(200).then(result => {
```

Comment:
For this I would suggest checking the review_id is the searched review_id (2) to ensure this returns the correct article.